docs/docs/administration/settings/signal/definition.mdx

```diff
@@ -0,0 +1,65 @@
+# Signal Definition
+
+Signal Defintions are your starting point for onboarding a new Signal into Dispatch and allow a number of different configuration options.
+
+## Sending Signals to Dispatch
+
+You can send a signal to Dispatch by making a `POST` request to the `/{organization}/signals/instances` API endpoint.
+
+```python
+import requests
+
+API_TOKEN = ""
+API_ENDPOINT = "http://localhost:8080/api/v1/default/signals/instances"
+
+signal_instance = {
+  "project": "Test",
+  "raw": {
+    "name": "process_events",
```

Comment:
```suggestion
    "name": "process_events",
    "externalId": "someExternalValue",
    "id": "<uuid>", # this will be used as the instance's uuid if provided
    "variant": "A" , # an additional indicator for signal subtypes
```

Reply:
Looks like this change was made